search-commons/src/main/java/no/unit/nva/search/SearchClient.java

```diff
@@ -48,7 +46,7 @@ public SearchResourcesResponse searchSingleTerm(Query query, String index)
         return toSearchResourcesResponse(query.getRequestUri(), query.getSearchTerm(), searchResponse.toString());
     }
 
-    public SearchResponse findResourcesForOrganizationIds(String index, UserResponse.ViewingScope viewingScope)
+    public SearchResponse findResourcesForOrganizationIds(String index, ViewingScope viewingScope)
```

Comment:
ViewingScope is a separate class, since it is not always connected to a UserResponse